compiler/ast_to_ir.py

```diff
@@ -106,8 +107,16 @@ def compile_asts(ast_objects, fileIdGen, config):
         # log("Compiling AST {}".format(i))
         # log(ast_object)
 
+        log("MMG config")
+        log(config['shell_variables'])
```

Comment:
If these are not needed we could remove them since the log would fill up with these calls to `ast_to_shell`. Maybe we could use the `level` argument of the `log` function to make them have a high debug level so that they do not appear when using `-d 1` ?